systemtest/src/main/java/io/strimzi/systemtest/resources/operator/BundleResource.java

```diff
@@ -116,15 +112,6 @@ public BundleResourceBuilder withExtraEnvVars(List<EnvVar> extraEnvVars) {
             return self();
         }
 
-        public BundleResourceBuilder defaultConfigurationWithNamespace(String namespaceName) {
-            this.name = Constants.STRIMZI_DEPLOYMENT_NAME;
-            this.namespaceInstallTo = namespaceName;
-            this.namespaceToWatch = this.namespaceInstallTo;
-            this.operationTimeout = Constants.CO_OPERATION_TIMEOUT_DEFAULT;
-            this.reconciliationInterval = Constants.RECONCILIATION_INTERVAL;
-            return self();
-        }
-
```

Comment:
I guess this is also good to have

Reply:
reverting back. Hopefully it will be used.